src/Diagnostics.ModelsAndUtils/Diagnostics.ModelsAndUtils.csproj

```diff
@@ -18,6 +18,7 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.2.0" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
     <PackageReference Include="System.Collections.Immutable" Version="1.3.1" />
+    <PackageReference Include="WindowsAzure.Storage" Version="9.3.3" />
```

Comment:
>WindowsAzure.Storage [](start = 31, length = 20)

This version of Storage SDK is not actively maintained any more. 

> NOTE: As of version 9.4.0, this library has been split into multiple parts and replaced:  See Microsoft.Azure.Storage.Blob, Microsoft.Azure.Storage.File, Microsoft.Azure.Storage.Queue, and Microsoft.Azure.Storage.Common.

You can switch to the newer libraries that are active maintained. The classes and the functions are pretty much the same.

Reply:
@puneetg1983  Yes, I tried using the newer library for table operations but I kept running into exceptions:
`Microsoft.Azure.Cosmos.Table.StorageException: Not Implemented
   at Microsoft.Azure.Cosmos.Table.RestExecutor.TableCommand.Executor.ExecuteAsync[T](RESTCommand`1 cmd, IRetryPolicy policy, OperationContext operationContext, CancellationToken token)`

`ErrorMessage:The requested operation is not implemented on the specified resource.`